cypress/e2e/po/pages/extensions-compatibility-tests/extensions-compatibility.po.ts

```diff
@@ -0,0 +1,98 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
```

Comment:
`ExtensionsCompatibiliyPo` is the parent class for `ElementalPo` or `KubewardenPo`. The whole logic for this is assembled around that. 
`ExtensionsCompatibiliyPo` accesses code from other specs and "makes" generalised to be accessed in this particular case of extensions (it also provides a couple of different functions I haven't found elsewhere), but the gist is that it's a "helper" class. 
To refactor all of this is to throw out https://github.com/rancher/dashboard/pull/11626/files#diff-907d7318234761519ab86650ff1329d9907618996ead924257cb08c303e13f05 and this PR.

Imho, I don't think it's that bad and given the amount of time I've already spent on this it would be quite an endeavour, which I am not inclined to do at this point. 

I am willing to discuss it in a call if you like. Let me know your thoughts. We can also bring @nwmac

Reply:
After discussed, i updated PR as required (see https://github.com/rancher/dashboard/pull/11508/commits/1dc44a5ec0cc68cf87c28dc23ab655bc5a94e947)